.github/workflows/builder-image-workflow.yml

```diff
@@ -0,0 +1,32 @@
+name: Build custom ledger-app-builder-bitcoin image
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+    paths:
+      - .github/workflows/builder-image-workflow.yml
+      - .github/workflows/Dockerfile
+
+jobs:
+  build:
+    name: Build and push ledger-app-builder image
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+    - name: Clone
+      uses: actions/checkout@v2
+
+    - name: Build and push ledger-app-builder-bitcoin to GitHub Packages
+      uses: docker/build-push-action@v1
+      with:
+        dockerfile: .github/workflows/Dockerfile
+        repository: ledgerhq/app-bitcoin-new/ledger-app-builder-bitcoin
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+        tag_with_sha: true
+        tags: latest
```

Comment:
Don't you want a specific tag stating weither the image comes from `develop` or `master` branch?